docs/blog/2019-04-18-component-shadowing/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Component Shadowing
+date: 2019-04-18
+author: John Otander
+excerpt: "Gatsby Themes introduce a concept called Component Shadowing. This feature allows
+users to override a component in order to customize its rendering."
+tags:
+  - themes
+---
+
+Gatsby Themes introduce a concept called Component Shadowing. This feature allows
+users to override a component in order to customize its rendering. Component Shadowing
+is a powerful way to make one off changes to a theme without having to fully eject.
```

Comment:
```suggestion
is a powerful way to make one-off changes to a theme without having to fully eject.
```

Made a grammar suggestion. Also, what does "having to fully eject" mean?

Reply:
Here, "fully eject" would refer to forking the theme to use as the base of your site. Eventually we'll also support a CLI command to automate this for folks that are making a lot of customizations to a theme.

Should we go more in-depth about what the term means in the context of themes (or maybe even remove that portion)?